pom.xml

```diff
@@ -330,6 +332,13 @@ under the License.
                 <requireMavenVersion>
                   <version>3.1.1</version>
                 </requireMavenVersion>
+                <requireProperty>
+                  <property>project.build.outputTimestamp</property>
+                  <message>The property "project.build.outputTimestamp" must be set on the reactor's root pom.xml to make the build reproducible. Further information at "https://maven.apache.org/guides/mini/guide-reproducible-builds.html".</message>
+                  <!-- https://www.myintervals.com/blog/2009/05/20/iso-8601-date-validation-that-doesnt-suck/ -->
+                  <regex>^([\+-]?\d{4}(?!\d{2}\b))((-?)((0[1-9]|1[0-2])(\3([12]\d|0[1-9]|3[01]))?|W([0-4]\d|5[0-2])(-?[1-7])?|(00[1-9]|0[1-9]\d|[12]\d{2}|3([0-5]\d|6[1-6])))([T\s]((([01]\d|2[0-3])((:?)[0-5]\d)?|24\:?00)([\.,]\d+(?!:))?)?(\17[0-5]\d([\.,]\d+)?)?([zZ]|([\+-])([01]\d|2[0-3]):?([0-5]\d)?)?)?)?$</regex>
```

Comment:
This does not cover because also a seconds since epoch are allowed. Processing will happily fail when the input format is wrong: https://maven.apache.org/shared/maven-archiver/xref/org/apache/maven/archiver/MavenArchiver.html#L775. I would go with the require only.

Reply:
Done in 45d63d2.